src/applications/personalization/profile/components/alerts/DirectDepositConnectionError.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
+
+import { benefitTypes } from '../direct-deposit/DirectDepositV2';
+
+const DirectDepositConnectionError = ({ benefitType }) => {
+  let headline;
+  let content;
+  switch (benefitType) {
+    case benefitTypes.CNP:
+      headline =
+        'We can’t load disability compensation and pension information';
```

Comment:
This definitely works!

We might be able to simplify a bit, like so, so that we don't need the `switch`:

```
  let type = benefitType === benefitTypes.EDU ? 'education' : 'disability compensation and pension';

  const headline = `We can’t load ${type}` information;

  const content = (
      <p>
        `We’re sorry. Something went wrong on our end. We are having trouble
        loading information about ${type}` benefits. Please refresh this page or try again later.
      </p>
    );
```

but that's just a matter of preference, I would also approve what you currently have up.

Reply:
Yeah I thought about doing it that way. I like that this way is more flexible and mirrors how I handled determining which message to show after a user has successfully updated their bank info. But it really can go either way.

Reply:
Yeah, totally works for me either way 💯